hash_practice/exercises.py

```diff
@@ -1,20 +1,101 @@
+from hash_practice.linkedlist import LinkedList
+
+
+# sums might be bb ac might add to same number. Consider prime hash in future #imlementation
+# split each word 
+# get ascii for each letter
+# add letters/ascii values together for each word
+# make that the key and add word to anagram/value list 
+# return anagram list for each key  
+
+def hash_helper(word): 
+    """ This method will takes a set of inputs of any arbitrary size and fits it into a table or other data structure that contains fixed-size elements.   
+        Time Complexity: O(n) checking each word
+        Space Complexity: O(1) rewriting to unicode_char
+    """ 
+    hash = 0
+    for letter in word:
+        unicode_char = ord(letter)
+        hash += unicode_char
+    return hash
 
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
-    """
-    pass
+    
+        Time Complexity: O(n) checking each word
+        Space Complexity: O(n) at worse case might have to create a node
+    """ 

```

Comment:
👍 , The time complexity is O(n) assuming the words are of limited length like real English words (no word is longer than so many characters) Otherwise it would be O(nm) where m is the word length.